src/openms/source/FORMAT/HANDLERS/TraMLHandler.cpp

```diff
@@ -627,15 +627,15 @@ namespace OpenMS::Internal
           os << "    <Peptide id=\"" << writeXMLEscape(it->id) << "\" sequence=\"" << it->sequence << "\">" << "\n";
           if (it->hasCharge())
           {
-            os << "      <cvParam cvRef=\"MS\" accession=\"MS:1000041\" name=\"charge state\" value=\"" <<  it->getChargeState() << "\"/>\n";
+            os << R"(      <cvParam cvRef="MS" accession="MS:1000041" name="charge state" value=")" <<  it->getChargeState() << "\"/>\n";
```

Comment:
Is there a reason not to "rawify" the  "\"/>\n" ?

Reply:
guess it's just awful to read with the newline and they decided against it?